### ISSUE TITLE
How to train from scratch(re-pretraining)?

### ISSUE THREAD

**Issue by eclipse0922** [uni-medical/STU-Net#28]
Thank you for sharing awesome work!

I did fine tuning for my dataset with pre-trained weight(small and base)
It shows me great result.

Can you share the training script used to train the TotalsSegmentator dataset?
Or is it sufficient to use the method in this link?
https://github.com/wasserth/TotalSegmentator/blob/master/resources/train_nnunet.md

I would like to add more data sets to the pre-training model to maximise performance.

like Verse and CTPelvic1K more,
Of course, I may have to change the information on their labels.

**Comment by eclipse0922** [comment]
same question.
https://github.com/uni-medical/STU-Net/issues/11
